handlers/templates/pages/guest-link-index.html

```diff
@@ -82,6 +86,50 @@
           });
       });
     });
+
+    document
+      .querySelectorAll('[aria-label="Enable"], [aria-label="Disable"]')
+      .forEach((toggleBtn) => {
+        toggleBtn.addEventListener("click", () => {
+          const id = toggleBtn.getAttribute("pico-link-id");
+          // switch enable/disable
+          if (toggleBtn.getAttribute("aria-label") === "Enable") {
+            guestLinkEnable(id)
+              .then(() => {
+                const button = toggleBtn;
+                button.classList.remove("is-danger");
+                button.classList.add("is-success");
+                button.setAttribute("aria-label", "Disable");
+                button.innerHTML =
+                  '<i class="fas fa-check-square" aria-hidden="true"></i>';
+                document
+                  .querySelector("snackbar-notifications")
+                  .addInfoMessage("Successfully disabled the guest link.");
```

Comment:
The snackbar messages are flipped, so it should say enabled here and disabled below.